python/lsst/rubintv/production/starTracker.py

```diff
@@ -62,7 +62,11 @@
 __all__ = (
     'StarTrackerWatcher',
     'StarTrackerChannel',
-    'StarTrackerNightReportChannel'
+    'StarTrackerNightReportChannel',
+    'StarTrackerCatchup',
+    'getCurrentRawDataDir',
+    'getDataDir',
+    'getFilename',
```

Comment:
Bubble up the last three for appearance-in-file ordering?

Reply:
Ah, I deliberately put them in the order they're defined in the code to make it easier to keep them correct. I wonder if that's not right now though. Thoughts?

Reply:
I'm I misreading something?  The first function I see below is `getCurrentRawDataDir`.  Anyway, I do leave it up to you how you'd like this ordered (the only guidance I see in the dev guide is https://developer.lsst.io/python/numpydoc.html#complete-example-module

Reply:
lol, you're not misreading anything, I'm just an idiot 🙃 Fixed.